activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -16,6 +16,12 @@ def self.find(version)
       V5_2 = Current
 
       class V5_1 < V5_2
+        module CommandRecorder
```

Comment:
You should move these changes to the `V5_2` class now that `V6_0` is current.

Reply:
Done.